Vagrantfile

```diff
@@ -9,7 +9,7 @@ Vagrant.configure('2') do |config|
     vb.customize ['modifyvm', :id, '--ioapic', 'on']
   end
 
-  config.vm.define :vagrant_demo_server do |service_config|
+  config.vm.define :rr_provision do |service_config|
```

Comment:
Please, consider this rename of the machine. I think that it need much better represent what machine is, than just say something like `vargant_demo_server`. Maybe we can choose better naming here.